mock/elrondProxyStub.go

```diff
@@ -0,0 +1,65 @@
+package mock
+
+import (
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/core"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+)
+
+// ElrondProxyStub -
+type ElrondProxyStub struct {
```

Comment:
Can use this to refactor all the test in the elrond client, when merged  I can take a look at it.

Reply:
Also my thinking again is that this is only in `elrond/client` and should not be needed by any other package, so it should live as close as possible to `elrond/client_test`

Reply:
when the monitor & relay will be refactored, I'm pretty sure they will need a proxy stub (Relay should not create inner components for test-ability purposes). Will remain in elrond-eth-bridge/mock package